README_nfs-injector-MagiskModule.md

```diff
@@ -0,0 +1,20 @@
+####Magisk module: NFS-Injector
+
+For those that might suffer with performance issues with their atv devices, for example, constant crashing, NFS Injector helps manage your ram and provides an overall better experience with performance. It will probably need more testing, but so far there are reports that even a 1gb model was able to scan for quests. 
+
+Not that the previous statment means that you should go and buy a ton of 1 gb models, because it's not recommended at all, some people just get sent the wrong models or have bad ram and this can help out so as your money was not a complete waste. However the device in question wouldn't even load the game properly before but yielded some results with the module installed.
+
+Another example, a 2gb model would freeze up completely and would require the power cord to be pulled out nearly every 10 minutes, now said device will run for hours before freezing up, if at all.
```

Comment:
Change this to:
Another example, some 2gb models would freeze up completely and would require the power cord to be pulled out nearly every 10 minutes. Now said device might run for hours or even stop freezing up completely.